mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -64,9 +77,14 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7258 isSpawnForced ()Z
 	METHOD method_7259 incrementStat (Lnet/minecraft/class_3445;)V
 		ARG 1 stat
+	METHOD method_7260 increaseRidingMotionStats (DDD)V
```

Comment:
I feel like `updateRidingMotionStats` would work a little better, but personal preference.

Reply:
I agree, however I went with this due to other places already using names like `createStat`

Reply:
This name is fine. The update is strictly incremental.

Reply:
Marking these as "resolved" then, as it applied to both this method and the travel variant below.